sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/StoreResult.java

```diff
@@ -208,6 +208,9 @@ public void serialize(StoreResult storeResult,
             jsonGenerator.writeObjectField("transportRequestTimeline", storeResult.storeResponse != null ?
                 storeResult.storeResponse.getRequestTimeline() :
                 storeResult.exception != null ? BridgeInternal.getRequestTimeline(storeResult.exception) : null);
+            jsonGenerator.writeObjectField("transportRequestChannelAcquisitionContext", storeResult.storeResponse != null ?
```

Comment:
Is there possibly a shorter name? "channelAcquisition"?

Reply:
Don't we need to hide it behind the new boolean check that you added ?